wordpress-mobile/WordPress-Android
# Save posts for later - Expand/collapse animation

Fixes #7719 

Adds collapse animation when removing an item from saved posts in the saved posts list and expand animation for undo.

To test:
1. Go to Reader - Saved posts list
2. Remove a post from saved posts
3. Notice the item transforms into an Undo item
4. Click on the item
5. Notice the item transforms back to the original item

![expand_collapse_animations](https://user-images.githubusercontent.com/2261188/40181169-bdb3303c-59e8-11e8-8200-e1d6f55d8312.gif)

cc @iamthomasbishop I haven't strictly followed your notes. This is default Android animation and I think it looks pretty good. Implementing our own animation would take extra time and I'm not sure it's worth it. wdyt?
